calcom/cal.com
# fix: Correctly prefill username in case of non-org email invite in an org

## What does this PR do?

- In case of non-org email(e.g. abc@gmail.com) when invited to an org, the signup page shows username as `abc` instead of `abc-gmail` - This issue is fixed now. It fixes the issue reported here https://github.com/calcom/cal.com/pull/12774#issuecomment-1860257694
- Also, username check wasn't considering the organization scope correctly and thus it was checking the username availability across all orgs - This is fixed now.

## Type of change

## How should this be tested?
See https://github.com/calcom/cal.com/pull/12774#issuecomment-1860257694 for more details.

## Mandatory Tasks
